### PR TITLE
SB-723: Remove unneeded GreetingProperties.

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

File: src/main/java/io/openshift/booster/service/GreetingEndpoint.java [hunk 1]
```diff
@@ -17,26 +17,16 @@
 
 import org.springframework.stereotype.Component;
 
-import javax.ws.rs.DefaultValue;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
```
Comment (gytis) [inline src/main/java/io/openshift/booster/service/GreetingEndpoint.java]: We don't normally use package import.

File: src/test/java/io/openshift/booster/OpenShiftIT.java [hunk 2]
```diff
@@ -16,16 +16,15 @@
 
 package io.openshift.booster;
 
-import java.net.URL;
```
Comment (gytis) [inline src/test/java/io/openshift/booster/OpenShiftIT.java]: No need to reorder.

File: src/main/java/io/openshift/booster/service/Greeting.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
 public class Greeting {
 
+    public static final String FORMAT = "Hello, %s!";
     private final String content;
```
Comment (gytis) [inline src/main/java/io/openshift/booster/service/Greeting.java]: We usually separate attributes with a new line.

**Comment by metacosm** [comment]
No, it’s the .codeeditor effect…

**Comment by metacosm** [comment]
@gytis the reformatting is due to .editorconfig taking effect, I think so we might need to tweak the settings more

**Comment by metacosm** [comment]
Unfortunately, it doesn't look like EditorConfig is powerful enough to specify all the rules we'd need. Another option would be to use Eclipse's code formatter since it's also available in IDEA…

**Comment by gytis** [comment]
Another option is to have checkstyle plugin. It's used in Spring Boot and I've used it in a couple other projects. But I don't think it's practical to add it to the boosters as it requires extra config in pom.xml as well as a separate config file.

I guess the easies option would be to agree on the same formatter settings (could share the eclipse one) and try to keep code inline during reviews. I don't think that we need to include it to every booster though. Simply hosting it in one place and asking everybody to setup their IDEs with it should be sufficient.

**Pull request merged**